lib/models/repository-states/present.js

```diff
@@ -528,9 +540,9 @@ export default class Present extends State {
           this.transitionTo('TooLarge');
           return {
             branch: {},
-            stagedFiles: [],
-            unstagedFiles: [],
-            mergeConflictFiles: [],
+            stagedFiles: {},
```

Comment:
just for my own education: why did you change this function to return objects instead of arrays?  (I'm sure there's a perfectly good reason, I just don't know what it is 😃 )

Reply:
heh, yeah good question. it's not immediately clear.

This was to fix an oversight. I checked the format of the `results` returned from this function and objects are expected rather than arrays. Probably doesn't make a difference for the error case here since they're all empty objects anyways, but consistency is good